src/olympia/devhub/templates/devhub/new-landing/index.html

```diff
@@ -1 +1,70 @@
-New DevHub Landing
+<!DOCTYPE html>
+<html lang="{{ LANG }}" dir="{{ DIR }}">
+  <head>
+    <meta charset="utf-8">
+      {% if not settings.ENGAGE_ROBOTS %}
+        <meta name="robots" content="noindex">
+      {% endif %}
+    {% block extrameta %}{% endblock %}
+    <title>{% block title %}{{ dev_page_title() }}{% endblock %}</title>
+
+    <link rel="shortcut icon" type="image/x-icon"
+          href="{{ STATIC_URL }}img/favicon.ico">
+
+    {% block rss_feed %}{% endblock %}
+
+    {# L10n: {0} is an application, like Firefox. #}
+    <link title="{{ _('{0} Add-ons')|f(APP.pretty) }}"
+          rel="search" type="application/opensearchdescription+xml"
+          href="{{ url('amo.opensearch') }}" />
+
+    {% block site_css %}
+      {{ css('devhub/css') }}
+    {% endblock %}
+
+    {% block extrahead %}{% endblock %}
+
+    {% if request.user.is_authenticated() %}
+      <meta name="csrf" content="{{ csrf_token }}">
+    {% endif %}
+
+    {{ js('preload') }}
+  </head>
+  <body class="html-{{ DIR }} {{ request.APP.short }} {% block bodyclass %}{% endblock %}"
+        data-app="{{ request.APP.short }}"
+        data-appname="{{ request.APP.pretty }}"
+        data-appid="{{ request.APP.id }}"
+        data-anonymous="{{ (not request.user.is_authenticated())|json }}"
+        data-readonly="{{ settings.READ_ONLY|json }}"
+        data-media-url="{{ MEDIA_URL }}"
+        data-static-url="{{ STATIC_URL }}"
+        data-raven-dsn="{{ settings.RAVEN_DSN }}"
+        data-raven-urls="{{ settings.RAVEN_ALLOW_LIST }}"
+        {% block bodyattrs %}{% endblock %}>
+
+  <section class="devhub-header">
+    <nav class="devhub-navigation">
+      <ul>
+        <li><a href="#"><img src="{{ STATIC_URL }}img/developers/addons-logo2.svg"></a></li>
+        <li><a href="#">Docs</a></li>
+        <li><a href="#">Support</a></li>
+        <li><a href="#">Blog</a></li>
+      </ul>
+    </nav>
+
+    <h1>{{ _('Customize Your Firefox') }}</h1>
+    <p>
+      {%- trans -%}
+      Add-ons let millions of Firefox users enhance their browsing experience.
+      If you know <a href="#">HTML</a>, <a href="#">JavaScript</a>, and <a href="#">CSS</a>,
```

Comment:
I would vote to keep `<a href="...">foo</a>` in the block-string tbh, these are strings that might need switching places in other languages and looking at `{{ html_link_start }}foo{{link_end}}` or anything similar we could run into problems with dennis-cmd linting not detecting this